### PR TITLE
Fix BE + FE validation duplications

### PR DESCRIPTION
**Pull request by PPiedel**

### CODE REVIEW

File: ui/client/components/graph/node-modal/editors/Validators.ts [hunk 1]
```diff
@@ -1,51 +1,65 @@
 import _ from "lodash"
 
 export enum validatorType {
+  FRONT_END, BACKEND,
```
Comment (lciolecki) [inline ui/client/components/graph/node-modal/editors/Validators.ts]: BACK_END :)

File: ui/client/components/graph/node-modal/editors/Validators.ts [hunk 2]
```diff
@@ -1,51 +1,65 @@
...
+  description: string,
+  handledErrorType: handledErrorType,
+  validatorType: validatorType,
 }
 
 export type Error = {
   fieldName: string,
   message: string,
   description: string,
+  typ: string,
```
Comment (lciolecki) [inline ui/client/components/graph/node-modal/editors/Validators.ts]: type: ValidatorType

File: ui/client/components/graph/node-modal/editors/Validators.ts [hunk 3]
```diff
@@ -1,51 +1,65 @@
 import _ from "lodash"
 
 export enum validatorType {
```
Comment (lciolecki) [inline ui/client/components/graph/node-modal/editors/Validators.ts]: ValidatorType

Comment (lciolecki) [inline ui/client/components/graph/node-modal/editors/Validators.ts]: ValidatorType

File: ui/client/components/graph/node-modal/editors/Validators.ts [hunk 4]
```diff
@@ -1,51 +1,65 @@
 import _ from "lodash"
```
Comment (lciolecki) [inline ui/client/components/graph/node-modal/editors/Validators.ts]: Can we remove using lodash?

File: ui/client/components/graph/node-modal/editors/Validators.ts (hunk 1 above)
Comment (PPiedel) [reply ui/client/components/graph/node-modal/editors/Validators.ts]: ...Frontend, Backend :)

File: ui/client/components/graph/node-modal/editors/Validators.ts (hunk 2 above)
Comment (PPiedel) [reply ui/client/components/graph/node-modal/editors/Validators.ts]: Nope, it comes from be as string

File: ui/client/components/graph/node-modal/editors/Validators.ts (hunk 3 above)
Comment (PPiedel) [reply ui/client/components/graph/node-modal/editors/Validators.ts]: Done

File: ui/client/components/graph/node-modal/editors/Validators.ts (hunk 4 above)
Comment (PPiedel) [reply ui/client/components/graph/node-modal/editors/Validators.ts]: Done

File: ui/client/test/ValidationLabels-test.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    const fieldName = "fieldName"
+    const emptyValue = ""
+    const backendErrorDescription = "test"
+    const backendError = {message: "Test", description: backendErrorDescription, typ: "EmptyMandatoryParameter", fieldName: "fieldName"}
+    const validators = [
+      mandatoryValueValidator,
+      errorValidator([backendError], "fieldName")
+    ]
+
+    render(<ValidationLabels validators={validators} values={[emptyValue]}/>)
```
Comment (alien11689) [inline ui/client/test/ValidationLabels-test.js]: where are `when` and `then` comments? :)

File: ui/client/test/ValidationLabels-test.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+    expect(screen.findAllByText(mandatoryValueValidator.description).length).toBe(1)
+    expect(screen.findAllByText(backendErrorDescription).length).toBe(0)
+  })
+
+  it("display validations for different error type", () => {
+    //given
+    const fieldName = "fieldName"
```
Comment (alien11689) [inline ui/client/test/ValidationLabels-test.js]: why don't you put common `const`s above tests?

File: ui/client/test/ValidationLabels-test.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
+import '@testing-library/jest-dom/extend-expect'
+import React from 'react'
+import {render, screen} from '@testing-library/react'
+import {errorValidator, mandatoryValueValidator} from "../components/graph/node-modal/editors/Validators"
+import ValidationLabels from "../components/modals/ValidationLabels"
+
+
+describe("displaying validation labels", () => {
+
+  test('display only fe validation when both be and fe validations available for the same error type', () => {
```
Comment (alien11689) [inline ui/client/test/ValidationLabels-test.js]: should it be `test` or `it`?

File: ui/client/test/ValidationLabels-test.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    expect(screen.findAllByText(mandatoryValueValidator.description).length).toBe(1)
+    expect(screen.findAllByText(backendErrorDescription).length).toBe(0)
+  })
+
+  it("display validations for different error type", () => {
+    //given
+    const fieldName = "fieldName"
+    const emptyValue = ""
+    const backendErrorDescription = "test"
+    const backendError = {message: "Test", description: backendErrorDescription, typ: "AnotherErrorType", fieldName: "fieldName"}
```
Comment (alien11689) [inline ui/client/test/ValidationLabels-test.js]: it is no easy to spot the difference between tests in this file... Maybe you can try to create these tests are parameterized?

File: ui/client/test/ValidationLabels-test.js (hunk 7 above)
Comment (PPiedel) [reply ui/client/test/ValidationLabels-test.js]: merged into one parametrized it test

File: ui/client/test/ValidationLabels-test.js (hunk 6 above)
Comment (PPiedel) [reply ui/client/test/ValidationLabels-test.js]: done

File: ui/client/test/ValidationLabels-test.js (hunk 5 above)
Comment (PPiedel) [reply ui/client/test/ValidationLabels-test.js]: done

File: ui/client/test/ValidationLabels-test.js (hunk 8 above)
Comment (PPiedel) [reply ui/client/test/ValidationLabels-test.js]: done

**Pull request merged**